pkg/types/types.go

```diff
@@ -9,6 +9,7 @@ import (
 type VfState struct {
 	HostIFName   string
 	SpoofChk     bool
+	Trust        bool
```

Comment:
if a pod started with an older version of this CNI (cmdAdd was called) it will not contain this field in cache.
then after upgrading sriov-cni, cmdDel is called, it will fail to unmarshall since Trust field will not be in the file.

does the above sound right ?

if so we should add a json tag/annotation to mark this as optional ("omitempty"), then it would default to false if it doesnt exist.

Reply:
great thinking!

But the `omitempty` works the other way around.
when we write the file not when we read the file. :)

https://www.sohamkamani.com/golang/omitempty/